source/client.c

```diff
@@ -131,21 +130,15 @@ static void s_mqtt_client_shutdown(
                 "id=%p: Discard ongoing requests and pending requests when a clean session connection lost.",
                 (void *)connection);
             aws_linked_list_swap_contents(&connection->synced_data.pending_requests_list, &requests);
+            aws_linked_list_move_all_back(&requests, &connection->thread_data.ongoing_requests_list);
```

Comment:
trivial: use move_all_back() in both places

also, add ongoing before pending so we can cancel in order

Reply:
trivial: rename `requests` to `cancelling_requests`